OlsztynHooligans/src/app/register/register.component.ts

```diff
@@ -1,19 +1,64 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
-
+import { IUser } from '../interface/interface.IUser';
+import { FormBuilder, FormGroup, NgForm, Validators } from '@angular/forms';
+import { AuthorizationDataService} from '../service/authorization-data.service';
 @Component({
   selector: 'app-register',
   templateUrl: './register.component.html',
   styleUrls: ['./register.component.scss']
 })
 export class RegisterComponent implements OnInit {
 
-  constructor(private router: Router) { }
+  constructor(private router: Router, private formBuilder: FormBuilder, private dataService: AuthorizationDataService) { }
+
+  user1: IUser;
+  userForm: FormGroup;
+  email: string;
+  password: string;
+  passwordconfirm: string;
 
   ngOnInit() {
+    this.user1 = { email: '', password: '', name: '', surname: '' };
```

Comment:
potrzebne to?

Reply:
użyłem tego aby na starcie w inputach nic nie wyświetlało